pkg/ds/daemon_set.go

```diff
@@ -187,7 +187,7 @@ func (ds *daemonSet) WatchDesires(
 					ds.CancelReplication()
 					continue
 				}
-				err := ds.removePods()
+				err = ds.removePods()
```

Comment:
Is this done just to save the allocation? I can't see any behavioural impact, but I want to be sure.